x/bank/types/keys.go

```diff
@@ -18,6 +18,9 @@ const (
 
 	// QuerierRoute defines the module's query routing key
 	QuerierRoute = ModuleName
+
+	// ModuleQueryPath defines the path of the module
```

Comment:
Maybe add that this would be used for abciqueries when not using the cli

Reply:
kk

Reply:
done